AntennaTracker/Parameters.cpp

```diff
@@ -250,8 +250,8 @@ const AP_Param::Info Tracker::var_info[] = {
     GOBJECT(compass,                "COMPASS_",     Compass),
 
     // @Group: SCHED_
-    // @Path: ../libraries/AP_Scheduler/AP_Scheduler.cpp
-    GOBJECT(scheduler, "SCHED_", AP_Scheduler),
+    // @Path: ../libraries/AP_Scheduler/AP_Scheduler.tpp
```

Comment:
Simple, because @dgrat removed GPLv3 statements and @tridge statement on AP_Scheduler.cpp.

Dear @dgrat your code style are seems good, but do you think that you are creating a new logic structure for Scheduler in AP_Scheduler.cpp?